pkgs/applications/editors/moe/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl
+, lzip, ncurses
+}:
+
+with stdenv.lib;
+stdenv.mkDerivation rec{
+
+  name = "moe-${version}";
+  version = "1.9";
+
+  src = fetchurl {
+    url = "mirror://gnu/moe/${name}.tar.lz";
+    sha256 = "1wsfzy0iia0c89wnx1ilzw54wqcmlp2nz8mkpvc393z0zagrx48q";
+  };
+
+  nativeBuildInputs = [ lzip ]
```

Comment:
Missing semicolon at end of line.

Reply:
Oh man, I was very distracted yesterday.